sklearn/_min_dependencies.py

```diff
@@ -40,6 +40,7 @@
     'numpydoc': ('1.0.0', 'docs'),
     'Pillow': ('7.1.2', 'docs'),
     'sphinx-prompt': ('1.3.0', 'docs'),
+    'dask': ('1.2.0', 'docs, examples'),
```

Comment:
Could you try to bump `scikit-image` to 0.14.5 specifically. This issue has been solved in the bug fixes our release but by specifying 0.14 we are indeed installing 0.14.0

Reply:
Will do ASAP